buildbot/master/init.sls

```diff
@@ -14,6 +14,7 @@ buildbot-master:
       - twisted == 16.6.0  # NOTE: keep in sync with buildbot-slave sls
     - require:
       - pkg: pip
+  {% if grains.get('virtual_subtype', '') != 'Docker' %}
```

Comment:
Why can't we get systemd running inside docker? Not having service.running seems not great, and certainly pollutes the states.